src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/UdtTest/SqlServerTypesTest.cs

```diff
@@ -252,7 +252,6 @@ private static void TestUdtSqlDataReaderGetStream(CommandBehavior behavior)
         }
 
         // Synapse: Parse error at line: 1, column: 41: Incorrect syntax near 'hierarchyid'.
-        [ActiveIssue("25421", TargetFrameworkMonikers.NetFramework)]
         [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup), nameof(DataTestUtility.IsNotAzureSynapse))]
         public static void TestUdtSchemaMetadata()
```

Comment:
Would you also mind removing `using System.Data.Common;" from this file please? There's also an extension method named `GetColumnSchema`, and I just want to sanity-check that we're testing `SqlDataReader.GetColumnSchema` rather than that.

Reply:
Done